plugins/modules/zos_started_task.py

```diff
@@ -0,0 +1,1639 @@
+        - Only applicable when I(state) is C(started) or C(modified), otherwise ignored.
+    required: false
+    type: list
+    elements: str
+#   retry_force:
+#     description:
+#         - Indicates whether retry will be attempted on ABTERM(abnormal termination).
+#         - I(tcb_address) is mandatory to use I(retry_force).
+#         - Only applicable when I(state) is C(forced), otherwise ignored.
+#     required: false
+#     type: bool
+  reus_asid:
+    description:
+        - When I(reus_asid) is C(True) and REUSASID(YES) is specified in the DIAGxx parmlib member, a reusable ASID is assigned
+          to the address space created by the START command. If I(reus_asid) is not specified or REUSASID(NO) is specified in
+          DIAGxx, an ordinary ASID is assigned.
+        - Only applicable when I(state) is C(started), otherwise ignored.
+    required: false
+    type: bool
+  state:
+    description:
+        - I(state) should be the desired state of the started task after the module is executed.
+        - If I(state) is C(started) and the respective member is not present on the managed node, then error will be thrown with C(rc=1),
+          C(changed=false) and I(stderr) which contains error details.
+        - If I(state) is C(cancelled), C(modified), C(displayed), C(stopped) or C(forced) and the started task is not running on the managed node,
+          then error will be thrown with C(rc=1), C(changed=false) and I(stderr) contains error details.
+        - If I(state) is C(displayed) and the started task is running, then the module will return the started task details along with
+          C(changed=true).
+    required: True
+    type: str
+    choices:
+        - started
+        - displayed
+        - modified
+        - cancelled
+        - stopped
+        - forced
+  subsystem:
+    description:
+        - The name of the subsystem that selects the task for processing. The name must be 1-4
+          characters long, which are defined in the IEFSSNxx parmlib member, and the subsystem must
+          be active.
+        - Only applicable when I(state) is C(started), otherwise ignored.
+    required: false
+    type: str
+  task_id:
+    description:
+        - The started task id starts with STC.
+        - Only applicable when I(state) is C(displayed), C(modified), C(cancelled), C(stopped), or C(forced), otherwise ignored.
+    required: false
+    type: str
+#   tcb_address:
+#     description:
+#         - 6-digit hexadecimal TCB address of the task to terminate.
+#         - Only applicable when I(state) is C(forced), otherwise ignored.
+#     required: false
+#     type: str
+#   volume:
+#     description:
+#         - If I(device_type) is a tape or direct-access device, the serial number of the volume,
+#           mounted on the device.
+#         - Only applicable when I(state) is C(started), otherwise ignored.
+#     required: false
+#     type: str
+  userid:
```

Comment:
Lets make it snake case, that way is more consistent with other option names.
```suggestion
  user_id:
```

Reply:
done